### PR TITLE
[-] FO : Fix bug TCPDF ERROR

### PR DESCRIPTION
**Pull request by Prestaspirit**
Fix bug TCPDF ERROR: [Image] Unable to get image

### CODE REVIEW

**Comment by gRoussac** [comment]
https://github.com/PrestaShop/PrestaShop/pull/2797#issuecomment-93458052

**Comment by gRoussac** [comment]
Hi,

Thank you for your PR.

- why not setting this for the admin too ? https://github.com/PrestaShop/PrestaShop/pull/2866/files#diff-39a9bfa5b8c16e7513adb810a173f21eL87

- I believe this may be required here too https://github.com/PrestaShop/PrestaShop/blob/1.6/classes/pdf/HTMLTemplate.php#L130-L132 before _PS_IMG_DIR_

Regards

File: classes/ImageManager.php [hunk 1]
```diff
@@ -86,7 +86,7 @@ public static function thumbnail($image, $cache_image, $size, $image_type = 'jpg
 		if (Context::getContext()->controller->controller_type == 'admin')
 			return '<img src="../img/tmp/'.$cache_image.($disable_cache ? '?time='.time() : '').'" alt="" class="imgm img-thumbnail" />';
 		else
-			return '<img src="'._PS_TMP_IMG_.$cache_image.($disable_cache ? '?time='.time() : '').'" alt="" class="imgm img-thumbnail" />';
+			return '<img src="'.Tools::getShopDomain(true)._PS_TMP_IMG_.$cache_image.($disable_cache ? '?time='.time() : '').'" alt="" class="imgm img-thumbnail" />';
```
Comment (jeromenadaud) [inline classes/ImageManager.php]: Hi,

Thank you for your contribution, what do you think about doing ```Context::getContext()->shop->getBaseURL(true, true)``` in order to have correct protocol ?

Best regards

**Comment by vTerenti** [comment]
(note: this is an automated message, but answering it will reach a real human )

Hi! Thank you for your pull request: your contribution to PrestaShop means a lot to us!

We are aware that this PR is still open after all this time, and we want to make sure that your hard work is not forgotten.

We are diving back into pull requests, and we want to work with you in order to get your PR to completion: we need your feedback!<br/>
Could you read [this article](http://build.prestashop.com/news/review-your-pull-requests/) from [the Build devblog](http://build.prestashop.com/), and take the time to review your pull requests, so that we can work with up-to-date information? You don't have to do it all right now, and every little step helps.

Again, thank you very much for this contribution, and for your renewed feedback on it!

**Comment by xBorderie** [comment]
Hey there!

This PR is fairly old and there hasn't been much activity on it. Closing, but please re-open it on the `develop` branch if the change is still relevant.

Thank you for your contribution!

**Pull request closed without merging**